notes.md

```diff
@@ -1,7 +1,7 @@
 # FireMon Security Manager
 Publisher: Splunk Community
 
-App Version: 1.0.0
```

Comment:
This file looks like same as README.md. can we remove it?

Reply:
sure, done.